### PR TITLE
Move image definitions into base-images/, trigger CI on it

### PR DESCRIPTION
**Pull request by cgwalters**
Only do image builds when the `base-images/` directory changes to avoid expensive CI runs for e.g. documentation PRs.

### CODE REVIEW

**Comment by cgwalters** [comment]
Hm, given that the Tekton didn't trigger on this PR perhaps the pathchanged isn't working

**Comment by lmilbaum** [comment]
I can work on limiting the number of builds triggered. Would you mind filing a Jira issue? 
BTW, I would prefer that GitHub pages files are tracked in the centos.github.io repo instead of here. It could provide us the ability to run CI on those doc files and even create a dev environment which spins a temporary website for manual testing.

**Comment by cgwalters** [comment]
> BTW, I would prefer that GitHub pages files are tracked in the centos.github.io repo instead of here.

But this project is not *itself* all of CentOS...

Think of this project a bit *kind* of like https://gitlab.com/CentOS/automotive/sample-images except it really is intending to be "version locked" (i.e. no separate kernels) and also larger in scope (datacenter, cloud, edge etc.)

**Comment by lmilbaum** [comment]
> > BTW, I would prefer that GitHub pages files are tracked in the centos.github.io repo instead of here.
> 
> But this project is not _itself_ all of CentOS...
> 
> Think of this project a bit _kind_ of like https://gitlab.com/CentOS/automotive/sample-images except it really is intending to be "version locked" (i.e. no separate kernels) and also larger in scope (datacenter, cloud, edge etc.)

I'm not sure I understand your feedback.  Is it related to the centos-boot github pages site?

**Comment by cgwalters** [comment]
> I'm not sure I understand your feedback. Is it related to the centos-boot github pages site?

Yes.  My understanding is you were suggesting we do something that is global to the centos namespace which I don't think we should do.

Maybe though we could create a centos-boot-docs GH repo; that seems perfectly fine to me.  Or I guess we could aim to land stuff in https://docs.centos.org/en-US/docs/

**Comment by lmilbaum** [comment]
> > I'm not sure I understand your feedback. Is it related to the centos-boot github pages site?
> 
> Yes. My understanding is you were suggesting we do something that is global to the centos namespace which I don't think we should do.
> 
> Maybe though we could create a centos-boot-docs GH repo; that seems perfectly fine to me. Or I guess we could aim to land stuff in https://docs.centos.org/en-US/docs/

IC where the confusion comes from. Tha centos-boot github page can stay the same as it is now. My point was to track it in a dedicated repo which GitHub maintains the deployment workflow and we have have a the flexibility to test it before changes land in the production environment.

**Comment by rhatdan** [comment]
Looks like this needs a rebase or to be closed.

File: .tekton/fedora-tier-1-eln-pull-request.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch && "base-images/**".pathChanged()
```
Comment (chmouel) [inline .tekton/fedora-tier-1-eln-pull-request.yaml:10]:
```suggestion
    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"&& "base-images/**".pathChanged()
```

and maybe remove line 11

**Comment by cgwalters** [comment]
Giant conflicts now, we'll just deal with the current situation

**Pull request closed without merging**